src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    return (price -= 50);
+  } else if (days >= 3) {
+    return (price -= 20);
+  }
+
+  return price;
```

Comment:
Consider using constants for the daily price, long term threshold, and discounts. This will make your code more readable and maintainable. For example, instead of using '40', '7', and '3' directly in the expressions, define constants at the beginning of your function with descriptive names.